### PR TITLE
Update BLorient795.xml

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

**Review by DenisNosnitsin1970 — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient795.xml [hunk 1]
```diff
@@ -32,8 +35,804 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                           <locus target="#24r"></locus>
+                           <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                               <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                           <q xml:lang="gez">ሲኖዶስ፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
+                     </q>
+                         </item>
+                       </list>
+                       <list>
+                         <item xml:id="e1">
+                           <desc>Name of the owner <persName ref="PRS13173WaldaK" role="owner"></persName>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient795.xml:779]: The names of...

**Review by thea-m — COMMENTED** [review]
I have a few remarks, but can come to it only on Thursday. Can the work not be identified?

**Comment by DariaElagina** [comment]
Whenever you have time. No the work can't be identified, there is no incipit or whatever.

File: LondonBritishLibrary/orient/BLorient795.xml [hunk 2]
```diff
@@ -32,8 +35,804 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                 <summary></summary>
+                 <msItem xml:id="p2_i1">
+                   <locus from="2r"></locus>
+                   <title type="complete" ref="LIT2461Testam" xml:lang="en">Testament
+                   of our Lord.</title>
+                   <textLang mainLang="gez"></textLang>
+
+                   <msItem xml:id="p2_i1.1">
+                     <locus from="2r"></locus>
+                     <title type="complete" xml:lang="en">Of the Holy Spirit.</title>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient795.xml:128]: Wright's chapters seem to correspond to the ones given in the work record. So you could maybe add the reference to all chapters in the title? And conversely, I would add the chapter titles in the work records, but that might be personal preference.

File: LondonBritishLibrary/orient/BLorient795.xml [hunk 3]
```diff
@@ -32,8 +35,804 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+               </msPart>
+               <msPart xml:id="p3">
+                 <msIdentifier></msIdentifier>
+                 <msContents>
+                   <msItem xml:id="p3_i1">
+                     <locus from="123r"></locus>
+                     <title type="incomplete" xml:lang="en">Theological work</title>
+                     <textLang mainLang="gez"></textLang>
+                     <note>The work was left unfinished by the scribe.</note>
+                     <note>The rubrics are partly lacking.</note>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient795.xml:803]: See above - if we assume that it's about rubrication, I would say that the extra is actually enough

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you for your patience!

**Review by thea-m — APPROVED** [review]

**Pull request merged**